__tests__/integration/chat/authenticated-persistence.contract.test.ts

```
@@ -0,0 +1,377 @@
+  };
+
+  const makeConversationUpdate = () => {
+    const filters: Record<string, any> = {};
+    let payload: any = null;
+
+    const builder: any = {
+      eq(field: string, value: any) {
+        filters[field] = value;
+        return builder;
+      },
+      then(
```

Biome (2.4.6)

[error] Do not add then to an object.

(lint/suspicious/noThenProperty)